tools/ecloud-revision-cleaner-tool/src/main/java/eu/europeana/cloud/jobs/RevisionRemoverJob.java

```diff
@@ -8,73 +8,154 @@
 import eu.europeana.cloud.mcs.driver.DataSetServiceClient;
 import eu.europeana.cloud.mcs.driver.RecordServiceClient;
 import eu.europeana.cloud.mcs.driver.RevisionServiceClient;
-import eu.europeana.cloud.mcs.driver.exception.DriverException;
 import eu.europeana.cloud.service.mcs.exception.MCSException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.RejectedExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
 import org.apache.log4j.Logger;
 import org.joda.time.DateTime;
 import org.joda.time.DateTimeZone;
 
-import java.util.Arrays;
 import java.util.Date;
 import java.util.List;
 
 /**
  * Created by Tarek on 7/15/2019.
  */
 public class RevisionRemoverJob implements Runnable {
```

Comment:
"implements Runnable" can be removed, and class name can be changed, because name job can be misleading